mobly/controllers/android_device.py

```diff
@@ -1053,27 +1053,50 @@ def take_bug_report(
     self.log.debug('Bugreport taken at %s.', full_out_path)
     return full_out_path
 
-  def take_screenshot(self, destination, prefix='screenshot'):
+  def take_screenshot(
+      self, destination, prefix='screenshot', all_displays=False
+  ):
     """Takes a screenshot of the device.
 
     Args:
       destination: string, full path to the directory to save in.
       prefix: string, prefix file name of the screenshot.
+      all_displays: bool, if true will take a screenshot on all connnected
+        displays, if false will take a screenshot on the default display.
 
     Returns:
-      string, full path to the screenshot file on the host.
+      string, full path to the screenshot file on the host, or list of full
```

Comment:
update the type hint in the beginning to avoid confusion.

now it can return a string or a list of strings.

maybe make a new line that starts with "list of strings"?